client/diagnostics_test/blob1.ion

```diff
@@ -0,0 +1 @@
+{{ 12345 }}
```

Comment:
maybe a better folder structure for tests should be 

`client/test/diagnostic/` 

I assume we will add more tests for the client that are not diagnostic. If we want to organize by type then a subfolder for each type of test seems cleaner

Reply:
The problem is that once I put the folder of highlighting test cases under another folder, the 'npm test' command will fail. Do we also need to change the pipeline setting files?

Reply:
See https://github.com/amzn/ion-vscode-plugin/blob/f447768a5a9793e83e894ed20abe3ff5afbed58b/package.json#L45

We should be able to create the folder structure that we want and direct each testing tool (highlighting and vscode) to the appropriate test folder to get its inputs

Reply:
Sure, now I know how to make the change.